### DISCUSSION TITLE
Support Woodpecker v1

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#23569] [Suggest an Idea]
Woodpecker v1 has changed the pipeline/workflow syntax. As a result, the woodpecker manager is not able to detect step images anymore. 

[v0.15.x ](https://woodpecker-ci.org/docs/0.15/usage/pipeline-syntax#image) syntax:

```
pipeline:
  backend:
    image: golang
``` 

[v1.0.x ](https://woodpecker-ci.org/docs/usage/pipeline-syntax#image) syntax:

```
steps:
  backend:
    image: golang
```

The woodpecker manager would need some adjustments to support both.

Reproduction repo: https://github.com/xoxys/renovate-woodpecker

**Comment by rarkins** [comment]
Converted to feature request issue #23574
